test/core/xds/xds_client_test.cc

```diff
@@ -2771,7 +2777,6 @@ TEST_F(XdsClientTest, AdsReadWaitsForHandleRelease) {
   resource1->read_delay_handle.reset();
   EXPECT_EQ(stream->reads_started(), 1);
   resource2->read_delay_handle.reset();
-  EXPECT_EQ(stream->reads_started(), 2);
```

Comment:
Please restore this line.  We want to check that another read has been started at this point, and then check it again later to make sure the next read hasn't been started yet before we trigger it.

Reply:
Done